lib/tasks/sync_kubernetes_secrets.rake

```diff
@@ -0,0 +1,57 @@
+namespace :kubernetes do
+  desc "Synchronise OAuth Token secrets in Kubernetes"
+  task :sync_token_secrets, %i[config_map_name] => :environment do |_, args|
+    client = Kubernetes::Client.new
+    config_map = client.get_config_map(args[:config_map_name])
+
+    emails = JSON.parse(config_map.data["api_user_emails"])
+    missing_users = []
+
+    emails.each do |email|
+      api_user = ApiUser.find_by(email: email)
```

Comment:
Suggest hoisting this query out of the loop (e.g. https://gist.github.com/robinjam/5279a5cfe3345823e46857510b961fed)